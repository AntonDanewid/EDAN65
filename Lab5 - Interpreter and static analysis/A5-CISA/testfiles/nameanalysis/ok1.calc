int main(int b) {
	return a(); /* okay: a refers to the parameter */
}

int a() {
	return 1;
}