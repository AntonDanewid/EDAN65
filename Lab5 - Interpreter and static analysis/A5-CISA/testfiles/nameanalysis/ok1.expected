Error at line 2: function 'a' is not declared
